adapters/conversant/conversant.go

```diff
@@ -18,6 +19,11 @@ type ConversantAdapter struct {
 }
 
 func (c *ConversantAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	cnvrRequest := *request
```

Comment:
Why are making a shallow copy of the request?

Reply:
I didn't want to affect any of the other adapters when I changed the top level currency field for my request

Reply:
ok, I removed the shallow copy

Reply:
Just checking in.  Are there any further changes needed?